src/fileEditor.cpp

```diff
@@ -43,17 +53,15 @@ FileEditor::FileEditor(const std::string& path)
 		!((active_perms & fs::perms::others_read) != fs::perms::none)
 	) {
 		endwin();
-		printf("Can't read the file, not enough permissions!\n");
+		std::cout << (std::string("Can't edit " + fullFilename + " not enough permissions. ").c_str());
```

Comment:
could've done `std::cout << "Can't edit " << fullFilename << " not enough permissions. ";`